addons/stock/views/stock_picking_views.xml

```diff
@@ -367,7 +367,8 @@
                         domain="[('state', 'in', ('assigned', 'waiting', 'confirmed')), '|', '|', ('has_deadline_issue', '=', True), ('date_deadline', '&lt;', current_date), ('scheduled_date', '&lt;', current_date)]"/>
                     <filter string="Planning Issues" name="planning_issues" help="Transfers that are late on scheduled time or one of pickings will be late"
                         domain="['|', ('delay_alert_date', '!=', False), '&amp;', ('scheduled_date','&lt;', time.strftime('%Y-%m-%d %H:%M:%S')), ('state', 'in', ('assigned', 'waiting', 'confirmed'))]"/>
-                    <filter string="Late Availability" name="late" domain="[('products_availability_state', '=', 'late')]"/>
+                    <filter string="Full Availability" name="full_availability" domain="[('products_availability_state', '=', 'available')]" invisible="context.get('is_receipt')"/>
```

Comment:
Why is it limited to receipt?

Reply:
@amoyaux Applying component availability in the receipt doesn't make sense because we are receiving goods, not selling them, so we don't need to check in receipts whether the component is available in stock or not.